Source/BTNavigationDropdownMenu.swift

```diff
@@ -222,7 +226,7 @@ public class BTNavigationDropdownMenu: UIView {
         self.backgroundView.alpha = self.configuration.maskBackgroundOpacity
         
         // Animation
-        UIView.animateWithDuration(self.configuration.animationDuration, delay: 0.15, options: UIViewAnimationOptions.TransitionNone, animations: {
```

Comment:
The spring animation looks great. But If I use spring animation, how can I change bounce offset of spring animation? And how can I make use of existing `bounceOffset` property?